lib/rust/flexer/src/automata/nfa.rs

```diff
@@ -206,7 +206,7 @@ impl From<&NFA> for DFA {
             let has_name = |&key:&state::Identifier| nfa.states[key.id].name.is_some();
             if let Some(eps) = epss.into_iter().find(has_name) {
                 let rule = nfa.states[eps.id].name.as_ref().cloned().unwrap();
```

Comment:
Why dont you just namem this `code` and use simpler constructor in the next line?